charts/quickwit/values.yaml

```diff
@@ -306,22 +306,27 @@ bootstrap:
 # Quickwit configuration
 config:
   # Metastore configuration.
-  postgres: {}
+  # Could be set either with QW_METASTORE_URI or with the following config:
+  # - If on S3:
+  # metastore_uri: s3://quickwit-indexes
+  # - If on postgres-like db:
+  # postgres: {}
   #   host: ""
   #   port: 5432
   #   database: metastore
   #   username: quickwit
   #   password: ""
 
-  s3: {}
```

Comment:
If I understand correctly, now the entire storage config is copy pasted into the quickwit.yaml of each node?
If so, can we put the config documentation link from our website?

Reply:
that's right, let's do this